scripts/load_testing/create_account.py

```diff
@@ -32,7 +32,11 @@ def signup(self):
 
         # capture email confirmation link on resulting page
         dom = pyquery.PyQuery(resp.content)
-        link = dom.find('#confirm-now')[0].attrib['href']
+        link = dom.find("a[href*='confirmation_token']")[0].attrib['href']
+        # FIXME: Use this when
+        # https://github.com/18F/identity-idp/commit/59e34df07f4e4f531b97331a91f159c0a709fcb2
+        # is deployed.
+        # link = dom.find('#confirm-now')[0].attrib['href']
```

Comment:
I think it's fine to keep this new change. No need to revert it back to using the id later, so I would remove the FIXME.